src/views/inspector/InspectorViewProvider.ts

```diff
@@ -213,6 +214,16 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
                 ${variableKeysInFeature.join('')}
               </div>
             </div>
+            ${environmentStatuses ? `<input id="collapsible-environment-status" class="toggle" type="checkbox" checked>
```

Comment:
why not just include the collapsible header in the getFeatureEnvironmentStatuses method so you can just insert `environmentStatuses` without the ternary?

Reply:
it'll return empty string if the environments isn't an object, but you can also just check if there are no environments in the object to output and just return early as well with an empty string